tests/app/main/test_briefs.py

```diff
@@ -2013,9 +2073,8 @@ def test_nice_to_haves_heading_not_shown_when_there_are_none(self, data_api_clie
         ) == 0
 
     def test_supplier_details_shown(self, data_api_client):
```

Comment:
It is, good spot, sorry I missed that. I had a look and it turns out that test can't be shared because it relies on different frameworks content depending on the flow.

I put in a fix for this in the latest commit (https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/593/commits/3f4053fad03ae8337141a278976b957fe6aa1538)